### ISSUE TITLE
Code does not work on the new Facebook

### ISSUE THREAD

**Issue by Huy-Nguyen-HCI** [dhruvramani/Terminal-on-FB-Messenger#20]
The following error is generated after logging in

```
File "main.py", line 230, in init
    replyButton = [x for x in driver.find_elements_by_tag_name('input') if x.get_attribute('value') == 'Reply'][0]
```

This might be because Facebook has changed its HTML layout.

**Comment by dhruvramani** [comment]
Yes, Facebook has modified how you chat. It's shifted to messenger.

You can contribute if you want, it shouldn't be a lot of work. I might not be able to contribute to for some time due to other work going on now. Hope you understand.
